doc/developers/contributing.rst

```diff
@@ -196,6 +196,17 @@ Contributing code
   so far. In order to claim an issue for yourself, please comment exactly
   ``take`` on it for the CI to automatically assign the issue to you.
 
+Video resources:
+----------------
+These videos are step-by-step introductions on how to contribute to scikit-learn, and are a great companion to the following text guidelines.
+Please make sure to still check our guidelines below, since they describe our latest up-to-date workflow.
```

Comment:
please make sure that the line length  is <79 characters ;)

Reply:
Sure!
So sorry for these petty mistakes...

Reply:
No worries at all, it's normal. thanks for your contributions!